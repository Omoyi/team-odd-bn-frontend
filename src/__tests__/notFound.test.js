import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../NotFound';


describe('Test Not Found page', () => {
  it('should render', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
  });
});
