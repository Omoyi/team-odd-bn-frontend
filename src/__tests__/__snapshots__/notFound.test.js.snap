// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Not Found page should render 1`] = `ShallowWrapper {}`;
